### ISSUE TITLE
Numix-based themes and mate-dock-applet

### ISSUE THREAD

**Issue by cellularlupus** [themix-project/oomox#181]
In versions of oomox newer than 1.6.0, the mate-dock-applet uses the background color for windows rather than the mate-panel background color when using numix-based themes. 

Here's a screenshot of the mate-dock-applet with a numix-based theme exported from oomox 1.6.0
![omoox 1 6 0](https://user-images.githubusercontent.com/38230298/50122553-0d35f500-022b-11e9-8d68-2a11d408bda5.png)

and here's a screenshot of the mate-dock-applet with a numix-based theme exported from oomox 1.6.2
![oomox 1 6 2](https://user-images.githubusercontent.com/38230298/50122578-38204900-022b-11e9-9bf9-33426b353604.png)

I've tried enabling the "change panel color to match wallpaper" and "change color of dock's panel only" settings in the mate-dock-applet's preferences to see if that makes a difference, but the background remains the same. 

mate-dock-applet is version 0.87
mate-desktop is version 1.20.3
i'm using ubuntu 18.10

**Comment by actionless** [comment]
could you please bisect on which version/commit that behavior got changed?

**Comment by actionless** [comment]
and also usually panels using the same color as CSD/menu while on your screenshot i see you're changing window background color

**Comment by cellularlupus** [comment]
i went through each commit between 1.6.1 and 1.6.2 and the behavior started with this commit https://github.com/themix-project/oomox/commit/32ea1029b762afe3a15b27ec2242871e7d0f189f

**Comment by cellularlupus** [comment]
![screenshot at 2018-12-18 15-49-31](https://user-images.githubusercontent.com/38230298/50182181-a53fe700-02dc-11e9-8344-337775ea9623.png)

**Comment by actionless** [comment]
yeah, it looks as intended, on other DE's panel is also themed the same as menubar/CSD

**Comment by actionless** [comment]
hint:
you can generate separate theme just for mate-panel and run mate-panel as `GTK_THEME=theme_name mate-panel`

**Comment by actionless** [comment]
or your point was not about panel itself but about the area at the left side of the panel, with some icons?

**Comment by actionless** [comment]
could you get with gtk inspector to that part of the panel to see what is it's layout? because in the commit you are pointing the only meaningful change was restyling of GtkViewport (and the result on your screenshot looks related to this change)

but it's quite unusual to put it on some panel since it's usually used for some scrollable areas

so what i gonna do in order to solve this issue, to add a style definition for case when GtkViewport is on the panel, so for that i need to know the widget layout of the panel

**Comment by cellularlupus** [comment]
the widget that's showing this behavior is the mate-dock-applet which can be found here https://github.com/robint99/mate-dock-applet

i've tried using gtk inspector but i think mate's panel works different than other desktop environments. when running the panel with the gtk inspector, the only parts of the panel i can select are separators, launcher icons, and areas of the panel with no widgets. 

i ran mate-panel with the GTK_THEME variable set to a dark theme with a dark grey menu/toolbar background, and with my system theme set to the one in my last comment and here's what it looks like:
![screenshot at 2018-12-18 17-04-17](https://user-images.githubusercontent.com/38230298/50186237-35cff480-02e8-11e9-8176-bf90cd04379f.png)

**Comment by actionless** [comment]
and if you selecting some, let's say, separator on the panel and go to CSS nodes what it the top level node?

![2018-12-18--1545172739_923x606_scrot](https://user-images.githubusercontent.com/1655669/50187353-24a1da80-031e-11e9-9d63-65a65ec4d9bd.png)

**Comment by actionless** [comment]
i also came up with alternative approach, instead of adding exception i made selector for viewport more specific

please check if it helps

**Comment by actionless** [comment]
[oomox_1.7.2.3-8-gc6f3e342_18.10+.deb.gz](https://github.com/themix-project/oomox/files/2692888/oomox_1.7.2.3-8-gc6f3e342_18.10%2B.deb.gz)

**Comment by actionless** [comment]
on CSS tab of inspector try adding:

```css
    #PanelApplet viewport {
background-color: #00ff00;
}
```
